src/app.js

```diff
@@ -31,6 +31,7 @@ export const canvas = new window.p5(sketch => { // eslint-disable-line
   const startBalls = () => {
     let id = 0
     const peopleWithAppInPercentage = document.getElementById('app_users')
+    const appFailurePercentage = document.getElementById('app_failures').value
```

Comment:
Consider to use `appFailuresElement` you defined in `dom.js`.
I know the other variables around it still use `document.getElementById(...)` even if defined in `dom.js`, but we should stop this practice someday :)